gradle.properties

```diff
@@ -1,2 +1,3 @@
 ltrVersion = 1.3.0
-elasticsearchVersion = 7.7.1
+elasticsearchVersion = 7.8.0
+luceneVersion = 8.5.0
```

Comment:
this is 8.5.1 (ref: https://github.com/elastic/elasticsearch/blob/v7.8.0/buildSrc/version.properties)